daemon/apparmor_default.go

```diff
@@ -26,6 +27,11 @@ func DefaultApparmorProfile() string {
 
 func ensureDefaultAppArmorProfile() error {
 	if apparmor.HostSupports() {
+		if !aaprofile.CanInstall() {
+			logrus.Warn("AppArmor enabled on system but apparmor_parser is missing, so profile can't be loaded")
```

Comment:
I don't think this error message should assert exactly what the issue is as there's two layers of indirection for the actual `LookPath` check. If we want to have informative error messages, perhaps it is better to early-return a non-fatal error if the binary is missing during subsequent attempts to actually invoke AppArmor?

Reply:
Moved the check to the `LoadProfile`.